src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
```

Comment:
What if no arguments are provided?
The app would throw an error. You should check if the arguments are provided and if they are of the right type.